gramex/apps/nlg/body.html

```diff
@@ -0,0 +1,344 @@
+            <i class="fa fa-plus-square"></i>
+        </button>
+    </p>
+</form>
+<div class="divider"></div>
+{% if dsid %}
+{% from os.path import basename %}
+{% set user_dir = basename(get_user_dir(handler)) %}
+<div class="formhandler" data-src="{{ nlg_base }}/preview/{{ user_dir }}/{{ dsid }}"></div>
+
+{% end %}
+<!--
+<button id="clear-tmpls" title="Clear Narrative" type="button" class="btn btn-danger">
+    <i class="fa fa-trash"></i>
+</button>
+-->
+<div class="modal fade" id="share-modal" role="dialog">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <pre>
+              <code class="language-html" id="share-narrative-url"
+              class="text- monospace"></code>
+            </pre>
+            <button id="copy-narrative-url" title="Copy To Clipboard"
+            type="button" class="btn btn-primary">
+                Copy Embed Code
+                <i class="fa fa-copy"></i>
+            </button>
+        </div>
+        </div>
+    </div>
+</div>
+<div class="modal fade" id="template-settings" role="dialog">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header"><h6>Template Settings</h6></div>
+            <div id="tmpllist" class="modal-body">
+                <div>
+
+                  <div class="card shadow mb-3">
+                    <div class="card-body">
+                      <h4 class="card-title">Template Preview</h4>
+                      <p class="card-text" id="tmpl-setting-preview"></p>
+                    </div>
+                  </div>
+
+                  <div class="form-group row">
+                    <label class="col-sm-4 col-form-label"
+                    for="tmpl-name-editor">Template Name</label>
+                    <div class="col-sm-4">
+                      <input class="form-control"
+                      id="tmpl-name-editor" placeholder="Name the template">
+                    </div>
+                    <div class="col-sm-4">
+                      <button id="tmpl-name-btn" type="button" class="btn btn-primary success">
+                          <i class="fa fa-check"></i>
+                      </button>
+                    </div>
+                  </div>
+
+                  <div class="form-group row">
+                    <label class="col-sm-4 col-form-label"
+                    for="condition-editor">Condition</label>
+                    <div class="col-sm-4">
+                      <input class="form-control"
+                      id="condition-editor" placeholder="Add condition">
+                    </div>
+                    <div class="col-sm-4">
+                      <button id="condition-btn" type="button" class="btn btn-primary success">
+                          <i class="fa fa-check"></i>
+                      </button>
+                    </div>
+                  </div>
+
+                  <div class="custom-control custom-switch mb-3">
```

Comment:
Remove this functionality entirely. Set all formhandler arguments in the generated template by default.